pandevice/base.py

```diff
@@ -2285,11 +2311,12 @@ class ParamPath(object):
         exclude (bool): Exclude this param from the resultant XML.
 
     """
-    def __init__(self, param, path=None, vartype=None,
+    def __init__(self, param, path=None, vartype=None, exist_tag=None,
```

Comment:
The way 'exist' used to work, it would use the 'path' as the tag that needs to exist.  Was that not working in this situation?  How come `exist_tag` is needed?